index.mjs

```diff
@@ -7,6 +7,15 @@ if (typeof process !== 'undefined') {
   global.Worker = function() { return { postMessage: function() { } } }
 }
 
+function kebabToCamel(attribute) {
+  if (attribute.includes('-')) {
+      return attribute.split('-').map((word, index) => index !== 0 ? word.charAt(0).  toUpperCase() + word.slice(1) : word
```

Comment:
There's a space between `charAt(0).` and `toUpperCase()`. Related: should we write a test for this? 😅

Reply:
I pushed a change to remove those spaces.

I debated adding unit tests for this function. I tested a bunch of permutations locally and really wondered if adding unit tests for something that would never change would be wasteful or not. Happy to add them but not 100% convinced of the utility here.

Reply:
Not a must for me either way! I just figured it might be useful in case the methodology or formatting changes in a breaking way in future (e.g. an extra space) ;) But totally yr call, may not be needed!

Reply:
Added tests